.github/scripts/sketch_utils.sh

```diff
@@ -121,16 +121,16 @@ function build_sketch(){ # build_sketch <ide_path> <user_path> <path-to-ino> [ex
     if [ -n "$ARDUINO_BUILD_DIR" ]; then
         build_dir="$ARDUINO_BUILD_DIR"
     elif [ $len -eq 1 ]; then
-        # build_dir="$sketchdir/build"
-        build_dir="$HOME/.arduino/build.tmp"
+        build_dir="$sketchdir/build"
+        # build_dir="$HOME/.arduino/build.tmp"
```

Comment:
different solution is needed here. It really is not nice to run CI locally and generate all those files in your  local fork. Maybe generate the artifacts elsewhere?

Reply:
OK will try to make it different way :)

Reply:
Is it ok now? Should be done the way you proposed to.

Reply:
yup. that is better :)